osc-domain/src/main/java/org/osc/core/broker/model/entities/virtualization/k8s/Label.java

```diff
@@ -37,13 +37,10 @@
 @Entity
 @Table(name = "LABEL")
 public class Label extends BaseEntity implements ProtectionEntity {
-    public Label() {
-    }
-
-    @Column(name = "value", nullable = false, unique = true)
+    @Column(name = "value", nullable = false)
     private String value;
 
-    @Column(name = "name", nullable = false, unique = true)
+    @Column(name = "name", nullable = false)
```

Comment:
i would have expected db upgrade step to change based on these fields being unique...for future make sure the annotation and the db matches.